test/e2e/service_file_test.go

```diff
@@ -124,11 +125,19 @@ func serviceCreateFromFile(r *test.KnRunResultCollector, serviceName, filePath s
 func serviceCreateFromFileError(r *test.KnRunResultCollector, serviceName, filePath string) {
 	out := r.KnTest().Kn().Run("service", "create", serviceName, "--filename", filePath)
 	r.AssertError(out)
-	assert.Check(r.T(), util.ContainsAllIgnoreCase(out.Stderr, "no", "such", "file", "directory", filePath))
+	assert.Check(r.T(), fileNotFoundErrorCheck(out, filePath))
 }
 
 func serviceCreateFromFileNameMismatch(r *test.KnRunResultCollector, serviceName, filePath string) {
 	out := r.KnTest().Kn().Run("service", "create", serviceName, "--filename", filePath)
 	r.AssertError(out)
 	assert.Check(r.T(), util.ContainsAllIgnoreCase(out.Stderr, "provided", "'"+serviceName+"'", "name", "match", "from", "file"))
 }
+
+func fileNotFoundErrorCheck(out test.KnRunResult, filePath string) cmp.Comparison {
+	result := util.ContainsAllIgnoreCase(out.Stderr, "no", "such", "file", "directory", filePath)
+	if result() == cmp.ResultSuccess {
```

Comment:
I wonder if that shouldn't be rather `== windows` check.

Reply:
I added a comment for clarity

Reply:
OK, thanks!